src/opentimelineio/safely_typed_any.cpp

```diff
@@ -59,6 +63,10 @@ int64_t safely_cast_int64_any(any const& a) {
     return any_cast<int64_t>(a);
 }
 
+uint64_t safely_cast_uint64_any(any const& a) {
+    return any_cast<uint64_t>(a);
+}
+
```

Comment:
@meshula I left the uint64_t types in the Any code.  Do you think it would be better to remove them entirely?  (It was missing this specific cast)

Reply:
I think it's harmless to have them as this won't affect things at the serialization/deserialization level.